rpc/params/ssh.go

```diff
@@ -58,3 +58,19 @@ type SSHPublicKeysResult struct {
 	Error      *Error   `json:"error,omitempty"`
 	PublicKeys []string `json:"public-keys,omitempty"`
 }
+
+// SSHHostKeyRequestArg provides a hostname to request the hostkey for.
+type SSHHostKeyRequestArg struct {
+	Hostname string `json:"hostname"`
+}
+
+// SSHHostKeyResult returns the host key for the target hostname.
+// Additionally, it returns the controller's SSH jump server's host key.
+//
+// We return the jump server's host key as to SSH to this unit, clients MUST
+// jump through the controller.
+type SSHHostKeyResult struct {
```

Comment:
could we names this PublicSSHHostKeyResult..

i'd also propose having a separate PublicSSHHostKey type that holds the two public keys and the result type that holds the error and the public ssh host key..

Reply:
I honestly just followed how it's done elsewhere. So I'll leave it as in for now. Renaming to your suggestion though.